front/lib/providers.ts

```diff
@@ -54,6 +54,14 @@ export const modelProviders: ModelProvider[] = [
     chat: true,
     embed: false,
   },
+  {
+    providerId: "mistral_ai",
```

Comment:
I'm wondering if we want `mistral_ai` or just `mistral` the `_ai` does not seem to serve any purpose given how unambiguous "mistral" is?

Reply:
Updated all occurrences to go with `mistral`.